engine/src/main/java/org/terasology/engine/rendering/dag/RenderGraph.java

```diff
@@ -52,8 +52,8 @@ public void addNode(Node node) {
         }
         if (akaNodeMap.containsKey(nodeAka)) {
             Node aNode = akaNodeMap.get(nodeAka);
-            logger.info("Node {} also known as {} already matches existing node with uri {} - attempting replacing...",
-                    nodeUri, nodeAka, aNode.getUri());
+            logger.atInfo().addArgument(nodeUri).addArgument(nodeAka).addArgument(() -> aNode.getUri()).
+                    log("Node {} also known as {} already matches existing node with uri {} - attempting replacing...");
```

Comment:
simple getter method
```suggestion
            logger.info("Node {} also known as {} already matches existing node with uri {} - attempting replacing...",
                    nodeUri, nodeAka, aNode.getUri()); //NOPMD
```